### ISSUE TITLE
关于bert.is_dropout参数的理解

### ISSUE THREAD

**Issue by XuJianzhi** [Tongjilibo/bert4torch#64]
bert = bert4torch.models.build_transformer_model
如果bert.is_dropout是默认的False，可否认为bert模型的hidden_dropout和attention_dropout都未生效？即，预训练模型的config.json的 attention_probs_dropout_prob 和 hidden_dropout_prob 参数实则为0 ？

有这样的疑问是因为，我对 hidden_dropout_prob 调参，出来结果不变。

class Model(BaseModel):
    def __init__(self, pool_method='cls') -> None:
        super().__init__()
        self.pool_method = pool_method
        self.bert = build_transformer_model(config_path=config_path, checkpoint_path=checkpoint_path, with_pool=True, model=model_type)
        self.dropout = nn.Dropout(dropout_rate)
        self.dense = nn.Linear(self.bert.configs['hidden_size'], 2)
        ########    改bert模型的dropout
        self.bert.dropout_rate = dropout_rate_hid

**Comment by Tongjilibo** [comment]
您好，第一个设想应该不是这样的，is_dropout=False并不会影响config里面的设置，只是会影响FeedForward中的dropout(可能我需要重新命名防止歧义)，你调参的话，一种方式是直接修改config来，另一种是build_transformodel时候传入dropout_rate， 你模型实例化后出来再修改属性是不行的

**Comment by XuJianzhi** [comment]
明白~但为什么模型实例化后出来再修改属性是不行？

**Comment by XuJianzhi** [comment]
再次感谢~
